test/renaissance/users/users_test.exs

```diff
@@ -1,40 +1,49 @@
 defmodule Renaissance.Test.UsersTest do
   use Renaissance.DataCase
-  require Ecto.Query
-  alias Renaissance.Users
-  alias Renaissance.Repo
+  alias Renaissance.{Repo, User, Users}
 
-  @valid_attrs %{email: "mail@mail.com", password: "password"}
+  @attrs %{email: "mail@mail.com", password: "password"}
 
-  describe "users" do
+  describe "register_user/2" do
     test "stores valid" do
-      {:ok, user} = Users.register_user(@valid_attrs)
+      {:ok, user} = Users.register_user(@attrs)
 
-      assert user.email == @valid_attrs.email
-      assert user.password_hash != @valid_attrs.password
+      assert Repo.exists?(User) == true
```

Comment:
`Repo.exists?(User)` will test that there is any user in the users table. I'm not sure if that's what's intended here. If so, a more clear and specific assertion would be nice.